### PR TITLE
Added ten nouns with three actions each

### PR DESCRIPTION
**Pull request by rallevondalle**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,84 +53,110 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 2:
+//  Noun: lavalamp
+//  action 1: lavalamp is turned ON
+lavalamp.on();
+//  action 2: lavalamp rotates
+lavalamp.rotate();
 
-//  action 3:
+//  action 3: lavalamp plays music
+lavalamp.music();
```
Comment (matkatmusic) [inline main.cpp]: the action is `plays`
the pseudo code needs to say this.
`music` is not an action, it is a noun.
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -52,84 +53,110 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 2:
-
-//  action 3:
+//  Noun: bicycle
+//  action 1: the bicycle rides
+bicycle.ride();
+//  action 2: the bicycle flies
+bicycle.fly();
+//  action 3: the bicycle dings its bell
+bicycle.bells();
```
Comment (matkatmusic) [inline main.cpp]: the action is `dings`
the pseudo code needs to say this.
`bells` is not an action, it is a noun.
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -52,84 +53,110 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bicycle
+//  action 1: the bicycle rides
+bicycle.ride();
```
Comment (matkatmusic) [inline main.cpp]: What is the bicycle riding?
Someone else rides the bicycle. 
how about `rolls`.?
"The bicycle rolls forward"
`bicycle.rollForward();`
- [x] revise

File: main.cpp [hunk 4]
```diff
@@ -52,84 +53,110 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
 //  5)
-//  Noun:
-//  action 1:
+//  Noun: Plants
+//  action 1: Plants needs watering
+plants.needsWatering();
 
-//  action 2:
+//  action 2: Plants photosynthesize
+plants.photosynthesis();
```
Comment (matkatmusic) [inline main.cpp]: `plants.photosynthesize();`
'synthesize' is the action.  'synthesis' is the noun.
your pseudocode needs to describe an action.
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

**Review by rallevondalle — COMMENTED** [review]
Corrected the errors.

**Comment by rallevondalle** [comment]
Changed some of the actions, wasn't the most creative ones I came up with last night haha :)

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**